docs/docs/Integrations/Composio/integrations-composio.mdx

```diff
@@ -7,14 +7,8 @@ Composio components in Langflow provide [Composio](https://app.composio.dev/) to
 
 Instead of juggling multiple integrations and components in your flow, connect Composio components to an **Agent** component to use all of Composio's supported APIs and actions as tools for your agent.
 
-The following components are available in the **Composio** bundle:
-
-* **Composio Tools**
-* **GitHub**
-* **Gmail**
-* **Google Calendar**
-* **Outlook**
-* **Slack**
+The Composio bundle includes a **Composio Tools** component that provides access to all Composio tools through a unified interface.
+Individual services like Gmail, GitHub, and Slack are also available as individual components, which authenticate and integrate with Composio the same way as the **Composio Tools** component.
```

Comment:
I would argue for listing out the component names to avoid confusion with legacy components like GMail Loader and GCal Loader.

I haven't rewritten this page yet, but I was going to go for something like the LangChain page where each component has a dedicated reference section: https://docs.langflow.org/bundles-langchain

You don't have to go to that length right now, but, as I said, I would at least list the name and purpose of each component.